appveyor.yml

```diff
@@ -1,10 +1,11 @@
 version: '{build}'
+image: Visual Studio 2015
 
 configuration:
   - RelWithDebInfo
 
 before_build:
-  - cmake -G "Visual Studio 10 2010" source/compiler -DCPACK_GENERATOR=ZIP
+  - cmake -G "Visual Studio 14 2015" source/compiler -DCPACK_GENERATOR=ZIP
```

Comment:
Any reason for updating to 2015? Visual Studio 2010 runtime was more widespread when this file was originally created so it was used to avoid troubles with missing MSVC runtime DLLs.

Reply:
The AppVeyor build failed because the new hashmap depends on `stdbool.h`, and VS 2010 didn't ship it. But maybe VS 2012 also has it, I didn't look into that.